test/services/ssg_config_downloader_test.rb

```diff
@@ -2,36 +2,68 @@
 
 require 'test_helper'
 
-# A class to test the SsgConfigDownloader service
+# A class to should the SsgConfigDownloader service
```

Comment:
:thinking:

Reply:
Mass replace of the word `test` to `should` :see_no_evil: